store/sql_store.go

```diff
@@ -611,3 +619,11 @@ func decrypt(key []byte, cryptoText string) (string, error) {
 
 	return fmt.Sprintf("%s", ciphertext), nil
 }
+
+// Interface for both gorp.DbMap and gorp.Transaction to allow code for one to be reused with the other
+type Queryable interface {
```

Comment:
It's internal now, but I figured it might be useful in the future if we ended up doing more things in transactions. And I can try just doing it with just a generic interface{}, but I'm not sure if we lose compile time type checking or anything that way.